api/bases/test.openstack.org_tempests.yaml

```diff
@@ -109,6 +109,10 @@ spec:
                 description: OpenStackConfigSecret is the name of the Secret containing
                   the secure.yaml
                 type: string
+              parallel:
```

Comment:
This option will be part of every CRD (Tobiko/Tempest/...). The parallel option works like this:

- If `parallel: false` then if test-operator is already executing other tests it will wait for the execution of the tests before starting a new set of tests.

- If `parallel: true` then if test-operator is already executing other tests then it will not wait for the execution of the tests to finish and it will start the tests straight away.

I can imagine that it is a little bit confusing without the documentation:/. I will update the documentation accordingly so that it is more clear.